index.fr.md

```diff
@@ -0,0 +1,111 @@
+---
+# Translation info https://www.w3.org/wiki/WAI/Website/Translate
+
+title: "Accessibilité mobile chez W3C"
+nav_title: "Accessibilité mobile chez W3C"
+
+description:
+
+lang: fr
+last_updated: 2020-09-03
+permalink: /standards-guidelines/mobile/fr
+
+translators:
+- name: "Alexander Akel"
+
+github:
+  repository: w3c/wai-mobile
+  path: index.fr.md
+
+feedbackmail: wai@w3.org
+footer: >
+  <p><strong>Date:</strong> Mise à jour le 03 September 2020. Première publication en janvier 2008.</p>
+  <p><strong>Éditeur:</strong> <a href="http://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.    
+  <p>Développé avec la contribution du Groupe de travail Éducation et Promotion (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+ref: /standards-guidelines/mobile/
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Sommaire" class="full" %}
+{:/}
+
+-   **L'accessibilité mobile est traitée dans les standards/règlements existant du W3C WAI**. Il n'y a pas de règles séparées spécifiques à l'accessibilité mobile.
+-   Le W3C est en train de mettre à jour les exigences et conseils spécifiques à l'accessibilité mobile.
+-   **Les efforts de la [force d’intervention sur l’accessibilité mobile](https://www.w3.org/WAI/GL/mobile-a11y-tf/) de la WAI se portent entre autres sur :**
+    -   [Accessibilité mobile : Comment WCAG 2.0 et les autres règles du W3C/WAI s'appliquent au mobile)](http://www.w3.org/TR/mobile-accessibility-mapping/)
+    -   WCAG 2.1 Critères de succès et techniques
+    -   Intégrer l'accessibilité mobile dans les standards W3C de demain
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Contenu de la page" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Plus que "Mobile" {#intro}
+
+"L'accessibilité mobile" fait référence au travail visant à rendre les sites web et applications plus accessibles aux personnes souffrant de handicaps lorsqu'ils utilisent des téléphones mobiles et autres appareils. Le travail de la WAI dans ce domaine traite des problèmes d'accessibilité pour les personnes utilisant une large gamme d'appareils pour interagir avec le Web incluant :
+
+-   téléphones et tablettes
+-   télévisions numériques
+-   appareils portables tels que les montres connectées
+-   appareils dans les tableaux de bord de voitures ou dossiers de sièges d'avion
+-   appareils dans l'électroménager
+-   autres "internet des objets"
+
+Cela couvre une grande étendue de problèmes concernant :
+
+-   écrans tactiles
+-   écrans de petite taille
+-   modes de saisie variés, incluant la voix et la technologie "3D Touch" facilitée par
+    des capteurs de pression
+-   appareils utilisés avec des paramètres différents comme les écrans lisibles en plein soleil (transflectifs)
```

Comment:
Keeping open for discussion

Reply:
@shawna-slh Can you help us better understand the original version ?

Reply:
to shed some light on: "device use in different settings, such as bright sunlight"
<pun intended :->

Indeed, we mean "environments", or even broader: "situations". (For example, I could be in a room without windows and the lights go out -- that would be a situational limitation, rather than an environmental limitation.)

I can update the English to be more clear, e.g.,:
"device use in different situations, such as in bright sunlight"
or maybe even better as:
"different situations, such as in bright sunlight"

Input and other suggestions welcome!

Reply:
Thank you @shawna-slh ! _"different situations, such as in bright sunlight"_ seems great.

@Alex-akel Can you update this PR accordingly ?

Reply:
I will update to 
"- l’utilisation d’appareils dans des situations différentes, par exemple en plein soleil"